cmd/tealdbg/local.go

```diff
@@ -158,10 +158,24 @@ type evalResult struct {
 
 // AppState encapsulates information about execution of stateful teal program
 type AppState struct {
-	appIdx  basics.AppIndex
-	schemas basics.StateSchemas
-	global  map[basics.AppIndex]basics.TealKeyValue
-	locals  map[basics.Address]map[basics.AppIndex]basics.TealKeyValue
+	appIdx       basics.AppIndex
+	schemas      basics.StateSchemas
+	global       map[basics.AppIndex]basics.TealKeyValue
+	locals       map[basics.Address]map[basics.AppIndex]basics.TealKeyValue
+	logs         []string
+	innerTxns    []transactions.SignedTxnWithAD
+	opcodeBudget int
```

Comment:
this should be in part of `cdtStateUpdate` and not `AppState`
Use it directly at
```
				appState := s.debugger.GetStates(&dbgState)
				state.Update(cdtStateUpdate{
					dbgState.Stack, dbgState.Scratch,
					dbgState.PC, dbgState.Line, dbgState.Error,
					dbgState. opcodeBudget,  <----
					appState,
				})
```